independent-publisher-connectors/VoiceRSS/apiDefinition.swagger.json

```diff
@@ -0,0 +1,148 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "VoiceRSS",
+    "description": "Voice RSS Text-to-Speech (TTS) allows conversion of textual content to speech easier than ever.",
+    "version": "1.0",
+    "contact": {
+      "name": "Fördős András",
```

Comment:
Please add URL as per the template here. https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/Independent%20Publisher/apiDefinition.swagger.json

Reply:
@vmanoharas I dont have a website URL to link, and as per documentation it is not mandatory
this can be also seen my other already existing IP connectors